core-tests/shared/src/test/scala/zio/duration/DurationSpec.scala

```diff
@@ -59,7 +60,7 @@ object DurationSpec
           },
           test("10 ns * NaN = Infinity") {
             assert(Duration.fromNanos(10L) * Double.NaN, equalTo(Duration.Infinity: Duration))
```

Comment:
Can you write up a quick summary of the issue here so we can raise an issue with the Dotty team if necessary?